content/content.go

```diff
@@ -13,7 +13,7 @@ import (
 var (
 	errNotFound = errors.New("content: not found")
 
-	bufPool = sync.Pool{
+	BufPool = sync.Pool{
```

Comment:
Fine with keeping it like this for now, just wanted to highlight it so we remember to figure it out later :)